host/sgx/create.c

```diff
@@ -874,6 +874,27 @@ oe_result_t oe_sgx_build_enclave(
         &loaded_enclave_pages_size,
         &enclave_size));
 
+    if (context->is_config_id_setting_mandatory && !(props.config.attributes & OE_SGX_FLAGS_KSS))
+    {
+        // Fail if the OE_SGX_FLAGS_KSS is not set when config_id/config_svn
+        // settings are mandatory
+        OE_RAISE_MSG(
+            OE_UNSUPPORTED,
+            "Enclave image requires config_id/config_svn settings but "
+            "OE_SGX_FLAGS_KSS is not set in properties\n",
+            NULL);
+    }
+
+    if(context->is_config_id_setting_mandatory && !_is_kss_supported())
+    {
+        // Fail if the CPU does not support KSS and cofig_id/config_svn setting are mandatory
```

Comment:
Typo:
```suggestion
        // Fail if the CPU does not support KSS and config_id/config_svn setting are mandatory
```

If you're using VSCode, consider using extensions like the [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to catch minor things like this.